python/pyspark/conf.py

```diff
@@ -60,11 +62,11 @@ class SparkConf:
 
     Examples
     --------
-    >>> from pyspark.core.conf import SparkConf
-    >>> from pyspark.core.context import SparkContext
+    >>> from pyspark import SparkConf
+    >>> from pyspark import SparkContext
```

Comment:
I changed this for the doctest to use the standard API usage while I am here. However, I can revert this change.